java/runtime/src/main/java/org/ray/runtime/RayNativeRuntime.java

```diff
@@ -72,11 +77,22 @@ private void resetLibraryPath() {
 
   @Override
   public void start() throws Exception {
-    // Load native libraries.
     try {
+      // Reset library path at runtime.
       resetLibraryPath();
```

Comment:
Do we still need this function to set `java.library.path`?

Reply:
yeah, because users may pass in custom library path.